ansible-scylla-node/tasks/Debian.yml

```diff
@@ -74,15 +69,91 @@
     when: scylla_version == 'latest' and scylla_edition == 'enterprise'
 
   - name: Install specified OSS Scylla
-    apt:
-      name: "scylla={{ scylla_version }}*"
-      state: present
+    block:
+      - name: Get dependencies of scylla package
+        shell: apt depends scylla={{ scylla_version }}*
+        register: aptdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scylladependencies: "{{ aptdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Get dependencies of scylla-tools package
+        shell: apt depends scylla-tools={{ scylla_version }}*
+        register: apttoolsdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scyllatoolsdependencies: "{{ apttoolsdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Get dependencies of scylla-server package
+        shell: apt depends scylla-server={{ scylla_version }}*
+        register: aptserverdependencies
+      - name: Set dependencies of scylla package as fact
+        set_fact:
+          scyllaserverdependencies: "{{ aptserverdependencies.stdout | regex_findall(regexp) }}"
+        vars:
+            regexp: ': ([a-z0-9+-\.]+) \('
+
+      - name: Install specified OSS Scylla packages
+        apt:
+          name: "{{ item }}={{ scylla_version }}*"
+          state: present
+        with_items: "{{ scyllaserverdependencies + scyllatoolsdependencies + scylladependencies }}"
+
+      - name: Install meta Scylla package
+        apt:
+          name: "scylla={{ scylla_version }}*"
```

Comment:
Why do you need to install a meta package? This will at least pull the latest scylla packages.
And in general - why does it need to be installed at all?

Reply:
Have you tested this PR on a Training Facility? Have you tried to install a version different from the latest? If yes, can I have a URL for that run?